tailwind.config.js

```diff
@@ -19,7 +19,7 @@ module.exports = {
     extend: {
       colors: {
         'gray': {
-          lighter: '#F6F6F6',
+          lighter: '#F7F7F7',
```

Comment:
Might want to confirm the value for this... I'm seeing `F3F3F3` in the figma files - a move to F7 would actually make it brighter, not darker.